github-pullrequest-plugin/src/main/resources/com/github/kostyasha/github/integration/branch/events/impl/GitHubBranchCommitMessageEvent/help.html

```diff
@@ -0,0 +1,8 @@
+<div>
+If set, the plugin will only build revisions commited with messages that match the pattern. If 'skip' is
+selected, any revisions with matching messages will be ignored.
```

Comment:
the initial idea of SKIP what that it SKIPS triggering for branch when any cause has `skip=true`